parsons/google/google_cloud_storage.py

```diff
@@ -158,6 +165,10 @@ def list_blobs(self, bucket_name, max_results=None, prefix=None, match_glob=None
         blobs = self.client.list_blobs(
             bucket_name, max_results=max_results, prefix=prefix, match_glob=match_glob
         )
+
+        if include_file_details:
+            return [b for b in blobs]
+
         lst = [b.name for b in blobs]
         logger.info(f"Found {len(lst)} in {bucket_name} bucket.")
```

Comment:
We should probably still do the logger statement in case we pass in `include_file_details`, right? I think it's weird that behavior would change just because we want to return more info about the blobs.

Reply:
Definitely, good catch